src/arena/BattleService.ts

```diff
@@ -46,17 +46,25 @@ function getTargetKeyboard(charId: string, game: Game, action: string) {
   if (!player) {
     throw new OrderError('Игрок не найден');
   }
+  const orders = game.orders.getPlayerOrders(charId);
   const { orderType } = getActions()[action];
   const proc = arena.skills[action] ? `_${arena.skills[action].proc}` : '';
+
   return game.players.alivePlayers
-    .filter((target) => (orderType === 'enemy' ? !game.isPlayersAlly(player, target) : true))
+    .filter((target) => {
+      if (orderType === 'enemy') {
+        return !game.isPlayersAlly(player, target);
+      }
+
+      return !orders.some((order) => target.id === order.target);
```

Comment:
Эта штука будет блокать заказы и на себя, т.е ты не сможешь дефать и лечить себя потому что цель заказа будешь ты сам, это не правильно. Нужно добавить исключение на тип действия "атака"

Reply:
Именно для атаки сделать исключение или может быть типа того? Т.е. любое действие чтобы можно было применить только один раз на одну цель?
```suggestion
      return !orders.some((order) => target.id === order.target && action === order.action);
```